lib/OpenStackExtract/MiqOpenStackVm/MiqOpenStackInstance.rb

```diff
@@ -0,0 +1,188 @@
+require 'util/miq_tempfile'
+require_relative '../../MiqVm/MiqVm'
+
+#
+# TODO: Create common base class for MiqOpenStackInstance and MiqOpenStackImage
+# and factor out common code. Also, refactor MiqVm so it can be a proper super class.
+#
+
+class MiqOpenStackInstance
```

Comment:
Only because all of the current MiqVm related classes are prefixed that way. Once this is refactored and generalized (all of the MiqVm stuff), we'll need to change it